cardano-node/src/Cardano/Node/Tracers.hs

```diff
@@ -217,16 +156,13 @@ mkTracers traceOptions tracer = Tracers
             logValue = LogValue "txsInMempool" $ PureI $ fromIntegral $ _txsInMempool mempoolEvent
         meta <- mkLOMeta Info Confidential
         traceNamedObject tr (meta, logValue)
-        case mempoolEvent of
-          TraceMempoolAddTxs      txs _ ->
-              let logValue' :: LOContent a
-                  logValue' = LogValue "txsProcessed" $ PureI $ fromIntegral $ length txs in
-              traceNamedObject tr (meta, logValue')
-          TraceMempoolRejectedTxs txs _ ->
-              let logValue' :: LOContent a
-                  logValue' = LogValue "txsProcessed" $ PureI $ fromIntegral $ length txs in
-              traceNamedObject tr (meta, logValue')
-          _                             -> return ()
+        let txs = case mempoolEvent of
+                  TraceMempoolAddTxs      txs0 _ -> txs0
+                  TraceMempoolRejectedTxs txs0 _ -> txs0
+                  _                             -> []
```

Comment:
My OCD would be happier if you align the last arrow with the two above :slightly_smiling_face: